mi-scheduler/base/openshift-templates/mi-run-template.yaml

```diff
@@ -13,6 +13,11 @@ parameters:
     description: GitHub repository whose Health Indicator will be computed
     displayName: inspected GitHub repository
 
+  - name: ENTITIES
+    required: true
```

Comment:
shouldn't be optional? If it is not provided, all entities are considered

Reply:
yes, that's actually true

Reply:
ugh, the remaining ``` lasted, weird. Consider this fixed